test/tests/functional/pbs_apbasil_inventory.py

```diff
@@ -224,3 +224,40 @@ def test_InventoryVnodes(self):
                 # Compare xml vnode with pbs node.
                 print "Validating vnode:%s" % (vnode['vnode'])
                 self.comp_node(vnode)
+
+    def test_cray_login_node_memory_unit(self):
+        """
+        This test validates that cray mom node resources value remain
+        unchanged before and after adding $alps_client in mom config.
+        """
+        mom_id = self.mom.shortname
+        try:
+            cray_login_node = self.server.status(NODE, id=mom_id)[0]
+        except PbsStatusError:
+            self.assertFalse(
+                "Cray login node %s doesn't exist on pbs server" % (mom_id))
+
+        self.mom.unset_mom_config('$alps_client', False)
+        self.reset_nodes(mom_id)
+
+        try:
+            pbs_node = self.server.status(NODE, id=mom_id)[0]
+
+        except PbsStatusError:
+            self.assertFalse(
+                "Cray login node %s doesn't exist on pbs server" % (mom_id))
+
+        for rsc, val in pbs_node:
+            if rsc in cray_login_node:
+                rval = cray_login_node[rsc]
+                if rval == val:
+                    self.logger.info(
+                        "%s\t: node has %s=%s" % (mom_id, rsc, rval))
+                    self.assertTrue(True)
```

Comment:
Why is this line required?

Reply:
This is just an indication of expected behavior.

Reply:
self.assertTrue(True) is a useless statement.  It will always pass.  If you want to say something to the test reader/runner, modify your log message.